src/model/transaction/insertTextIntoContentState.js

```diff
@@ -32,7 +32,11 @@ function insertTextIntoContentState(
     '`insertText` should only be called with a collapsed range.',
   );
 
-  const len = text.length;
+  let len;
+  if (text) {
+    len = text.length;
+  }
+
   if (!len) {
```

Comment:
I would also make this check more explicit as `len` can be `null` or `0`.
```js
if (len == null || len === 0) {
```
Note the non-strict equality check `len == null` that is also true if `len` is `undefined`.